README.md

```diff
@@ -88,56 +88,146 @@ by2ihzjyg9m674j3cjdit3reo  node-2    READY   ACTIVE
+ceuwhcffcavur7k9q57vqw0zg    redis      4           redis:3.0.5    RUNNING          RUNNING 3 seconds ago    node-1
+8vqmbo95l6obbtb7fpmvz522f    redis      5           redis:3.0.5    RUNNING          RUNNING 3 seconds ago    node-3
+385utv15nalm2pyupao6jtu12    redis      6           redis:3.0.5    RUNNING          RUNNING 3 seconds ago    node-3
+```
+
+Changing *instances* from *1* to *6* forced *SwarmKit* to create *5* additional Tasks in order to
+comply with the desired state.
+
+Every other field can be changed as well, such as image, args, env, ...
+
+Let's change the image from *redis:3.0.5* to *redis:3.0.6* (e.g. upgrade):
+
+```
+$ swarmctl service update redis --image redis:3.0.6
+89831rq7oplzp6oqcqoswquf2
+
+$ swarmctl service inspect redis
+ID                : 89831rq7oplzp6oqcqoswquf2
+Name              : redis
+Instances         : 6
+Template
+ Container
+  Image           : redis:3.0.6
+
+Task ID                      Service    Instance    Image          Desired State    Last State                Node
+-------                      -------    --------    -----          -------------    ----------                ----
+7947mlunwz2dmlet3c7h84ln3    redis      1           redis:3.0.6    RUNNING          RUNNING 34 seconds ago    node-3
+56rcujrassh7tlljp3k76etyw    redis      2           redis:3.0.6    RUNNING          RUNNING 34 seconds ago    node-1
+8l7bwrduq80pkq9tu4bsd95p4    redis      3           redis:3.0.6    RUNNING          RUNNING 36 seconds ago    node-2
+3xb1jxytdo07mqccadt06rgi0    redis      4           redis:3.0.6    RUNNING          RUNNING 34 seconds ago    node-1
+16aate5akcimsye9cp5xis1ih    redis      5           redis:3.0.6    RUNNING          RUNNING 34 seconds ago    node-2
+dws408a3gz0zx0bygq3aj0ztk    redis      6           redis:3.0.6    RUNNING          RUNNING 34 seconds ago    node-3
+```
+
+By default, all tasks are updated at the same time.
+
+This behavior can be changed by defining update options.
+
+For instance, in order to update tasks 2 at a time and wait at least 10 seconds between updates:
 
-Task ID                      Service    Instance    Image    Desired State    Last State              Node
--------                      -------    --------    -----    -------------    ----------              ----
-8oobrcr5u9lofmcbg7goaluyw    redis      1           redis    RUNNING          RUNNING 1 minute ago    node-1
 ```
+$ swarmctl service update redis --image redis:3.0.7 --update-parallelism 2 --update-delay 10s
+$ watch -n1 "swarmctl service inspect redis"  # watch the update
+```
+
+This will update 2 tasks, wait for them to become *RUNNING*, then wait an additional 10 seconds before moving to other tasks.
+
+Update options can be set at service creation and updated later on. If an update command doesn't specify update options, the last set of options will be used.
+
+### Node Management
+
+*SwarmKit* will monitor Node health and, in case of failure, re-schedule Tasks to other nodes.
```

Comment:
SwarmKit monitors node health. In the case of node failures, it re-schedules tasks to other nodes.

Node and Task are not proper nouns, lowercase.